corehq/apps/userreports/reports/builder/forms.py

```diff
@@ -223,33 +225,38 @@ def get_existing_match(self):
         ).one()
 
 
-class CreateNewReportForm(forms.Form):
-    """
-    A form for the first page of the report builder.
-    Allows user to specify report type, application, and source.
-    """
-    report_type = forms.ChoiceField(
```

Comment:
The report type is now determined by which button the user clicks in the `ReportBuilderTypeSelect` view, and `DataSourceForm` is initialized with a report type.